src/content/further-education/overview.mdx

```diff
@@ -0,0 +1,73 @@
+---
+order: 1
+parent: Further education
+title: Life Stage Overview
+bgImageName: further-education-bg.jpg
+redirect_from: 
+  - /issues-chronology/further-education/
+footnotes:
+  [
+    {
+      id: "footnote-1",
+      text: "Tom Sperlinger, Josie McLellan and Richard Pettigrew. (2018) Who are universities for? Re-making higher education. Bristol University Press, p. 42.",
+    },
+    {
+      id: "footnote-2",
+      text: "Social Market Foundation. (2017) Further education is key to social mobility in Brexit Britain.",
+    },
+    {
+      id: "footnote-3",
+      text: "Bridge Group. (2019) The importance of place: geographical isolation and progression to higher education.",
+    },
+  ]
+---
+
+import ResearchImage from "../../components/research/research-image";
+
+There has been a significant decline in adult participation in further education
+over the last ten years. Students from lower socio-economic backgrounds are
+overrepresented in lower level study and apprenticeships which have a weak
+return in the labour market in comparison to higher level qualifications (level
+3+). The sector has suffered acutely due to funding cuts. Spending per student
+in further education is lower than that received by students in secondary school
+and higher education.
+
+<div>
+  {/* FIXME missing graph */}
```

Comment:
Is this coming soon?

Reply:
yeah, I couldn't find it in the assets folder. Will add it in once Carina provides the graphs